lighthouse-core/report/html/report-styles.css

```diff
@@ -1470,7 +1504,6 @@
   display: block;
   animation: fadeInTooltip 250ms;
   animation-fill-mode: forwards;
-  animation-delay: 850ms;
```

Comment:
this removes this everywhere, we want that right? I would guess so because a lot of folks don't seem to know that they can hover over the error to get the tooltip 👍

Reply:
yeah, I figured it doesn't make sense to inject 850ms of latency for this.